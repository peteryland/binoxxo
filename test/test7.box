3o2o2o2
5x4xx
1o
6o4o
2o1ox2x1x
7o2x
o10o
1x1x1o3x
3x
x4x4x
1o
6o3x
