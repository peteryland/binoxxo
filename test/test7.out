___O__O__O__
_____X____XX
_O__________
______O____O
__O_OX__X_X_
_______O__X_
O__________O
_X_X_O___X__
___X________
X____X____X_
_O__________
______O___X_

OXOOXXOXXOOX
OOXOOXXOXOXX
XOXXOOXXOXOO
XXOOXXOXOXOO
OXOXOXOOXOXX
XOXOXOXOXOXO
OXXOXXOXOXOO
XXOXOOXOOXOX
OOXXOOXOXOXX
XXOOXXOXOOXO
OOXXOOXOXXOX
XOOXXOOXOXXO
